codes/quantum/qubits/gen_divisible.yml

```diff
@@ -0,0 +1,42 @@
+#######################################################
```

Comment:
There was just a new entry submitted for related codes:
https://errorcorrectionzoo.org/c/quantum_divisible
Any simple relation to these codes? They both seem to deal with divisibility, but that's already explained by cousin connection to divisible. If you can't see anything simple, nbd.